tools/generate-helpers/pg-table-bindings/list_test.go

```diff
@@ -16,3 +19,14 @@ func TestStorageToResource(t *testing.T) {
 	assert.Equal(t, "fake", storageToResource("storage.fake"))
 	assert.Equal(t, "fake", storageToResource("*storage.fake"))
 }
+
+func TestClusterGetter(t *testing.T) {
+	assert.Panics(t, func() { clusterGetter(walker.Walk(reflect.TypeOf(&storage.CVE{}), "")) })
+	assert.Equal(t, "obj.GetClusterId()", clusterGetter(walker.Walk(reflect.TypeOf(&storage.Deployment{}), "")))
+}
+
+func TestNamespaceGetter(t *testing.T) {
+	assert.Empty(t, namespaceGetter(walker.Walk(reflect.TypeOf(&storage.Email{}), "")))
+	assert.Empty(t, namespaceGetter(walker.Walk(reflect.TypeOf(&storage.Cluster{}), "")))
```

Comment:
Can you add a test when the search filed is not at the root of the proto structure ? e.g. storage.ProcessBaseline or storage.Risk ?